schema-admin.graphql

```diff
@@ -487,6 +487,97 @@ input CreateApprovedCuratedCorpusItemInput {
     scheduledSurfaceGuid: ID
 }
 
+"""
+Input data for loading an Approved Item via an automated process and optionally scheduling
+this item to appear on a Scheduled Surface.
+"""
+input LoadApprovedCuratedCorpusItemInput {
+    """
+    The GUID of the corresponding Prospect ID. Will be empty for manually added items.
+    """
+    prospectId: ID
+    """
+    The URL of the Approved Item.
+    """
+    url: Url!
+    """
+    The title of the Approved Item.
+    """
+    title: String!
+    """
+    The excerpt of the Approved Item.
+    """
+    excerpt: String!
+    """
+    The outcome of the curators' review of the Approved Item.
+    """
+    status: CuratedStatus!
+    """
+    What language this item is in. This is a two-letter code, for example, 'en' for English.
+    """
+    language: String!
+    """
+    The name of the online publication that published this story.
+    """
+    publisher: String!
+    """
+    The image URL for this item's accompanying picture.
+    """
+    imageUrl: Url!
```

Comment:
question: if we pass the `http://image_src` url, can the curatedCorpusApi fetch the byte stream and create a s3Url and return back with the response? 

This way, we can reduce two mutation calls to one and the server can take care of doing the node fetch to get the bytestream from the url

Reply:
I don't know, we could give it a try.

Reply:
We could implement the downloading --> creating a byte stream --> uploading to s3 on the back end/resolver logic.

Downloading from the `http` source is being done on the front end and the resolver for upload image just receives a byte stream. So I think half our work is already done on the backend